### PR TITLE
Add Nginx Proxy Manager icon

### LINKED ISSUE #7499: Request: Nginx Proxy Manager
### Brand Name

Nginx Proxy Manager

### Website

nginxproxymanager.com

### Popularity Metric

8.1k stars on GitHub, https://github.com/NginxProxyManager/nginx-proxy-manager

### Official Resources for Icon and Color

https://github.com/NginxProxyManager/nginx-proxy-manager/blob/develop/docs/.vuepress/public/logo.svg

### Additional Comments

_No response_

### Contributing

No

### PR DESCRIPTION
**Pull request by GiorgioReale**
nginxproxymanager

**Issue:** closes #7499

**Similarweb rank:** 249,403

### Checklist
  - [x] I updated the JSON data in `_data/simple-icons.json`
  - [x] I optimized the icon with SVGO or SVGOMG
  - [x] The SVG `viewbox` is `0 0 24 24`

### Description
This commit will resolve issue #7499 by adding Nginx Proxy Manager Logo in nginxproxymanager.svg and by updating simple-icons.json file

### CODE REVIEW

File: _data/simple-icons.json [hunk 1]
```diff
@@ -7777,6 +7777,11 @@
             "source": "https://www.nginx.com/press/",
             "guidelines": "https://www.nginx.com/press/"
         },
+        {
+            "title": "Nginx Proxy Manager",
+            "hex": "#000000",
```
Comment (PratyushJayachandran) [inline _data/simple-icons.json:7782]:
```suggestion
            "hex": "000000",
```
Remove #

**Comment by LitoMore** [comment]
Hi @jc21, do you have any preference for the icon's primary color?

Our library only supports single color since we store all icons with SVG and only have one path.

**Comment by GiorgioReale** [comment]
In the first commit of this PR, i had set the primary color as `#F15833` (the same as the website), but since I couldn't find any color guidelines, i decided to remove it.
I think it's a great idea to let @jc21 decide.

**Comment by jc21** [comment]
Hi, yeah `F15833` looks good to me. Thanks for the icon :)

**Comment by LitoMore** [comment]
I've pushed a commit to clean some redundant control points.

|Before|After|
|:-:|:-:
|||

**Review by LitoMore — APPROVED** [review]
Thank you @jc21 and @GiorgioReale!

**Pull request merged**